src/plugins/index_pattern_field_editor/public/components/field_editor/form_fields/popularity_field.tsx

```diff
@@ -10,5 +10,12 @@ import React from 'react';
 import { UseField, NumericField } from '../../../shared_imports';
 
 export const PopularityField = () => {
-  return <UseField path="popularity" component={NumericField} />;
+  // TODO how to pass props down to input for functional test
+  return (
```

Comment:
What we do is use chaining in our tests, so in this case it woulb be `editorFieldCount.input` to target the input.

If for some reason that does not work you could do this

```js
<UseField
      path="popularity"
      component={NumericField}
      componentProps={{
        euiFieldProps: { 'data-test-subj': 'editorFieldCount' }
      }}
    />
```